extensions/spark/kyuubi-spark-connector-tpcds/pom.xml

```diff
@@ -61,6 +61,13 @@
             <artifactId>guava</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-core_${scala.binary.version}</artifactId>
+            <type>test-jar</type>
```

Comment:
it's much better to build our own test framework

Reply:
#2738